pyresample/spherical.py

```diff
@@ -26,6 +26,18 @@
 
 logger = logging.getLogger(__name__)
 
+EPSILON = 0.0000001
+
+
+def _unwrap_radians(val, mod=np.pi):
+    """Put *val* between -*mod* and *mod*."""
+    return (val + mod) % (2 * mod) - mod
+
+
+def modpi(val, mod=np.pi):
+    """Put *val* between -*mod* and *mod*."""
+    return _unwrap_radians(val, mod=mod)
```

Comment:
Hopefully the last request: These functions are now exactly the same, right? If you think "unwrap radians" is a better name than "modpi" then delete "modpi". If not, delete `_unwrap_radians`. Replace whatever usage was deleted.

Reply:
@djhoese : `modpi` existed already so we need to keep it for backcompatibility. I think unwrap_radians is a better name ... what `modpi` stands for?

Reply:
`modpi` shouldn't be public API so no user should be using it (I know it doesn't have a `_` prefix but it isn't intended for public use). It stands for modulo based on pi (I assume).

Put another way, this is a private function inside this module and should *only* be used by this module. If it is used somewhere else then we should probably fix that.

Reply:
@djhoese done